OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -366,6 +366,7 @@
     <Compile Include="Traits\ProximityCapturable.cs" />
     <Compile Include="Traits\RadarColorFromTerrain.cs" />
     <Compile Include="Traits\Render\Hovers.cs" />
+    <Compile Include="Traits\Render\LeavesTrack.cs" />
```

Comment:
That is what I had in mind, but my feel for language wasn't good enough. Updated.